punchpipe/flows/level0.py

```diff
@@ -154,7 +165,7 @@ def level0_form_images(session=None, pipeline_config: str | dict | None = None):
                              'replay_length': image_packets_entries[-1].flash_block
                                               - image_packets_entries[0].flash_block + 1}
                     errors.append(error)
-            elif image_compression[0]['CMP_BYP'] == 1:
+            else:
```

Comment:
```suggestion
            elif image_compression[0]['JPEG'] == 1:
```

Looks like [`form_from_raw()`](https://github.com/punch-mission/punchpipe/blob/3f12a03c2e898853acf8ae437bf26210c0a0f682/punchpipe/level0/core.py#L405-L417) is hardcoded to 16bit pixels and thus wouldn't handle fully raw (19 bpp) images, so a check of this sort is still worthwhile IMO.